packages/eslint-plugin/tests/rules/no-parameter-properties.test.ts

```diff
@@ -13,6 +13,11 @@ class Foo {
 }
         `,
     `
+class Foo {
+    constructor(...name: string) {}
```

Comment:
(technically shouldn't this be
```suggestion
    constructor(...name: string[]) {}
```